apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/ast/CodeGenerationAst.kt

```diff
@@ -43,6 +43,7 @@ internal data class CodeGenerationAst(
   data class FragmentType(
       val graphqlName: String,
       val rootType: TypeRef,
+      val description: String,
```

Comment:
👍

Reply:
Once this is merged and used, I think we should try to upstream this inside the GraphQL spec so that other tools don't choke on it.